### ISSUE TITLE
1.0.6   if config_metric["metric_type"] not in valid_metric_types:

### ISSUE THREAD

**Issue by cicadadevops** [electrolux-oss/aws-cost-exporter#17]
i update config 

    image: opensourceelectrolux/aws-cost-exporter:v1.0.6

```

aws-cost-exporter_1  | Traceback (most recent call last):
aws-cost-exporter_1  |   File "//main.py", line 137, in <module>
aws-cost-exporter_1  |     validate_configs(config)
aws-cost-exporter_1  |   File "//main.py", line 85, in validate_configs
aws-cost-exporter_1  |     if config_metric["metric_type"] not in valid_metric_types:
aws-cost-exporter_1  | KeyError: 'metric_type'
```

**Comment by cicadadevops** [comment]
```
    metrics_type: AmortizedCost ->     metric_type: AmortizedCost
    add
        group_by:
      enabled: true


```

**Comment by gluckzhang** [comment]
Hi @cicadadevops, thanks for reporting this issue. Indeed, I refactored some code and now the configuration key becomes `metric_type` instead of `metrics_type`. I will update the example config file right now!

**Comment by gluckzhang** [comment]
The fix has been merged to the main branch from #18. Please adjust the config and keep using the `v1.0.6` image. Thanks and I am closing this issue!
